index.html

```diff
@@ -12,6 +12,34 @@
 </head>
 
 <body>
+
```

Comment:
The layout for the nav is almost complete.  There are a few flexbox properties you can use here to center the content on the page.  Also, putting the lambda-black.png in the `<nav>` may also assist you in getting the nav layout correct.